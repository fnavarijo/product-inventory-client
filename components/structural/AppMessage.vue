<template>
  <!-- TODO: Improve message -->
  <div class="text-center my-24">
    <span class="text-xl text-gray-800 font-bold">
      <slot />
    </span>
  </div>
</template>
