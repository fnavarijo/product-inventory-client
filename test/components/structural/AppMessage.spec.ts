import { mount } from '@vue/test-utils';

import AppMessage from '@/components/structural/AppMessage.vue';

describe('AppMessage component', () => {
  it('should generate message with proper style', () => {
    const wrapper = mount(AppMessage, {
      slots: {
        default: 'Something wrong happened',
      },
    });

    expect(wrapper.find('span').html()).toContain('Something wrong happened');
    expect(wrapper).toMatchSnapshot();
  });
});
