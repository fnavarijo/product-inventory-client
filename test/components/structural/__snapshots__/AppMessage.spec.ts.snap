// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppMessage component should generate message with proper style 1`] = `<div class="text-center my-24"><span class="text-xl text-gray-800 font-bold">Something wrong happened</span></div>`;
